docs/user-guide/pod-preset/index.md

```diff
@@ -0,0 +1,544 @@
+---
+assignees:
+- jessfraz
+title: Pod Preset
+---
+
+You can use a `podpreset` object to inject certain information into pods at creation
+time. This information can include secrets, volumes, volume mounts, and environment
+variables.
```

Comment:
Is it possible to extend it to apply also for pod/container resource limit/requests?